specification/eventgrid/data-plane/Microsoft.HealthcareApis/stable/2018-01-01/HealthcareApis.json

```diff
@@ -88,6 +88,60 @@
         }
       }
     },
+    "HealthcareDicomImageCreatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.HealthcareApis.DicomImageCreated event.",
+      "type": "object",
+      "properties": {
+        "dicomImageStudyInstanceUid": {
+          "description": "Unique identifier for the Study",
+          "type": "string"
+        },
+        "dicomImageSeriesInstanceUid": {
+          "description": "Unique identifier for the Series",
+          "type": "string"
+        },
+        "dicomImageSopInstanceUid": {
+          "description": "Unique identifier for the DICOM Image",
+          "type": "string"
+        },
+        "dicomServiceHostName": {
+          "description": "Domain name of the DICOM account for this image.",
+          "type": "string"
+        },
+        "dicomServiceSequence": {
+          "description": "Sequence number of the DICOM Service within Azure Health Data Services. It is unique for every image creation and deletion within the service.",
+          "type": "integer",
+          "format": "int64"
+        }
+      }
+    },
+    "HealthcareDicomImageDeletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.HealthcareApis.DicomImageDeleted event.",
+      "type": "object",
+      "properties": {
+        "dicomImageStudyInstanceUid": {
+          "description": "Unique identifier for the Study",
+          "type": "string"
+        },
+        "dicomImageSeriesInstanceUid": {
+          "description": "Unique identifier for the Series",
+          "type": "string"
+        },
+        "dicomImageSopInstanceUid": {
+          "description": "Unique identifier for the DICOM Image",
+          "type": "string"
+        },
+        "dicomServiceHostName": {
+          "description": "Host name of the DICOM account for this image.",
+          "type": "string"
+        },
+        "dicomServiceSequence": {
```

Comment:
Is the sequence purely a service concept? i.e. is not called out in the DICOM spec? Is this different than the Sequence defined here - https://dicom.nema.org/medical/Dicom/2016c/output/chtml/part03/sect_5.2.html

Reply:
I called this sequence based on your comment in the private PR. It was previously called `changefeedId`.

Its not the same sequence from that link. Its essentially the sequence number for changes occurred in the dicom service. Each dicom service keeps a change log. When any image is added or deleted from the service, that change gets added to the change log with a new sequence number. This change log is exposed through our Change Feed API https://github.com/microsoft/dicom-server/blob/main/docs/concepts/change-feed.md , which is specific to our service and not part of the DICOM standard.

I like the change to `sequence` because it is more generic, however I don't want to get it confused with this kind of Sequence https://dicom.nema.org/medical/Dicom/2016c/output/chtml/part03/sect_5.2.html . What do you think?

Reply:
Yeah, I think SequenceNumber is a bit less ambiguous even aside from the Dicom naming conflict.

Reply:
Yeah the Changefeed API is GA in our service, it was included in the initial GA release of the DICOM service in March.

I will go ahead and change to property to `sequenceNumber`